main.py

```diff
@@ -3,14 +3,27 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    while True:
+        # Event handling to check if the user wants to quit
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                return  # Exit the game loop if the user closes the window
```

Comment:
**Suggestion:** Ensure proper cleanup of Pygame resources when quitting.

**Existing Code:**
```Python
return  # Exit the game loop if the user closes the window
```

**Improved Code:**
```Python
pygame.quit()  # Clean up Pygame resources
return
```

**Details:**
Add a way to gracefully exit the game loop and clean up resources when quitting.